_posts/2019-08-01_Dossier_SafeCode_SlowSite

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "CLA Promotion and Tenure Dossier Builder tool, Creating a safe code review space, Linux Performance Metrics"
+date: 2019-08-01 09:30
+categories: upcoming
+---
+
+## Meta:
+
+- Location: [Walter Library](http://campusmaps.umn.edu/walter-library), Room 402 and on [Zoom](https://z.umn.edu/cpmstream)
+- Day: Thursday, Aug 1
+- Time: 9:30 - 11:00
+
+## Agenda:
+
+- 9:30 - Introductions
+- 9:35 - 9:55 [Demo CLA Promotion and Tenure Dossier Builder tool](#demo-cla-promotion-and-tenure-dossier-builder-tool), Elleni B Fellows
+- 9:55 - 10:20 [Creating a safe code review space](#creating-a-safe-code-review-space), Tony Thomas
```

Comment:
This talk is 15 min and Willy's talk is longer, can we adjust? Thanks!:

9:35 - Demo CLA Promotion and Tenure Dossier Builder Tool - (confirmed)
9:55 - Creating a Safe Code Review Space - (confirmed)
10:10 - Break
10:20 - Linux Performance Metrics - (confirmed)
10:45 Lightning Talks

Reply:
Thanks!  I adjusted the times.